tests/JsonSchema/Tests/Constraints/BaseTestCase.php

```diff
@@ -33,13 +40,18 @@ public function testInvalidCases($input, $schema, $checkMode = Validator::CHECK_
      */
     public function testValidCases($input, $schema, $checkMode = Validator::CHECK_MODE_NORMAL)
     {
+        $schema = json_decode($schema);
+
+        $refResolver = new RefResolver(new UriRetriever);
+        $refResolver->resolve($schema);
+
         $validator = new Validator($checkMode);
 
-        $validator->check(json_decode($input), json_decode($schema));
```

Comment:
I don't follow. Are you referring to the addition of `RefResolver` in the lines above or this specific line?